package.json

```diff
@@ -1,49 +1,49 @@
 {
-  "name": "starknet-spells-goerli",
-  "version": "0.0.1",
-  "description": "Starknet Spells Goerli",
-  "repository": "https://github.com/makerdao/starknet-spells-goerli",
-  "author": "Maciek Kaminski",
-  "license": "AGPL-3.0",
-  "scripts": {
-    "build": "tsc",
-    "chain:l1": "./scripts/node.sh",
-    "chain:l2": "./scripts/devnet.sh",
-    "chain": "./scripts/both.sh",
-    "format": "prettier --check \"test/**/*.ts\"",
-    "format:fix": "prettier --write \"test/**/*.ts\"",
-    "lint": "eslint --ext .ts src",
-    "lint:fix": "yarn lint --fix",
-    "test": "mocha --exit --recursive 'test/**/*.test.ts'",
-    "test:wait": "wget -q --retry-connrefused --delete-after http://0.0.0.0:5050/fork_status && yarn test",
-    "test:ci": "concurrently -k 'yarn devnet' 'yarn test:wait; docker stop devnet'"
-  },
-  "dependencies": {},
-  "devDependencies": {
-    "@nomicfoundation/hardhat-network-helpers": "^1.0.7",
-    "@nomiclabs/hardhat-ethers": "^2.2.1",
-    "@shardlabs/starknet-hardhat-plugin": "^0.7.1",
-    "@types/lodash": "^4.14.191",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.11.13",
-    "@typescript-eslint/eslint-plugin": "^5.46.1",
-    "@typescript-eslint/parser": "^5.46.1",
-    "@wagmi/core": "^0.8.15",
-    "chai": "^4.3.7",
-    "concurrently": "^7.6.0",
-    "earljs": "^0.2.3",
-    "eslint": "^8.29.0",
-    "eslint-config-typestrict": "^1.0.5",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-no-only-tests": "^3.1.0",
-    "eslint-plugin-simple-import-sort": "^8.0.0",
-    "eslint-plugin-sonarjs": "^0.17.0",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "ethers": "^5.7.2",
-    "hardhat": "^2.12.4",
-    "lodash": "^4.17.21",
-    "prettier": "^2.8.1",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.4"
-  }
+	"name": "starknet-spells-goerli",
+	"version": "0.0.1",
+	"description": "Starknet Spells Goerli",
+	"repository": "https://github.com/makerdao/starknet-spells-goerli",
+	"author": "Maciek Kaminski",
+	"license": "AGPL-3.0",
+	"scripts": {
+		"build": "tsc",
+		"chain:l1": "./scripts/node.sh",
+		"chain:l2": "./scripts/devnet.sh",
+		"chain": "./scripts/both.sh",
+		"format": "prettier --check \"test/**/*.ts\"",
+		"format:fix": "prettier --write \"test/**/*.ts\"",
+		"lint": "eslint --ext .ts src",
+		"lint:fix": "yarn lint --fix",
+		"test": "mocha --exit --recursive 'test/**/*.test.ts'",
+		"test:wait": "wget -q --retry-connrefused --delete-after http://0.0.0.0:5050/fork_status && yarn test",
+		"test:ci": "concurrently -k 'yarn devnet' 'yarn test:wait; docker stop devnet'"
+	},
+	"devDependencies": {
+		"@nomicfoundation/hardhat-network-helpers": "^1.0.7",
+		"@nomiclabs/hardhat-ethers": "^2.2.1",
+		"@shardlabs/starknet-hardhat-plugin": "^0.7.1",
+		"@types/chai": "^4.3.4",
```

Comment:
Do we want both chai and earljs? I don't have any preferences

Reply:
earljs is good